whisper: add page (#9901)
tldr-pages/tldr: pages/common/whisper.md

# whisper

> CLI tool to convert audio files to txt,vtt,srt,tsv,json.

> > Created By OpenAI. Read more:- https://github.com/openai/whisper .

Reviewer: ```suggestion
> Read more: <https://github.com/openai/whisper>.
```

> - Convert the audio file named audio.mp3 to all of the given file formats:

Reviewer 2: ```suggestion
- Convert a specific audio file to all of the given file formats:
```

> - Convert audio file named audio.mp3 to all of the given file formats.:

Reviewer: ```suggestion
- Convert audio file named audio.mp3 to all of the given file formats:
```

> `whisper audio.mp3`

Reviewer: ```suggestion
`whisper {{path/to/audio.mp3}}`
```

> - Specify only which file to generate:

Reviewer 3: ```suggestion
- Specify the output format of the converted file:
```

> - Specify the output format of the converted file:

Reviewer 2: ```suggestion
- Convert an audio file specifying the output format of the converted file:
```

> `whisper audio.mp3 --output-format txt`

Reviewer: ```suggestion
`whisper {{path/to/audio.mp3}} --output-format {{txt}}`
```

> - Specify which model to use for conversion:

Reviewer 2: ```suggestion
- Convert an audio file using a specific model for conversion:
```

> `whisper audio.mp3 --model tiny`

Reviewer: ```suggestion
`whisper {{path/to/audio.mp3}} --model {{tiny}}`
```

> - Specify which model to use (models:- tiny.en,tiny,base.en,base,small.en,small,medium.en,medium,large-v1,large-v2,large):
> 
> `whisper {{path/to/audio.mp3}} --model {{tiny}}`

Reviewer 3: ```suggestion
- Specify which model to use for conversion:

`whisper {{path/to/audio.mp3}} --model {{tiny.en,tiny,base.en,base,small.en,small,medium.en,medium,large-v1,large-v2,large}}`
```

> - Specify which language the audio file is to reduce time:

Reviewer 3: ```suggestion
- Specify which language the audio file is in to reduce conversion time:
```

> `whisper audio.mp3 --language english`

Reviewer: ```suggestion
`whisper {{path/to/audio.mp3}} --language {{english}}`
```

> - Specify which language the audio file is in to reduce conversion time:

Reviewer 2: ```suggestion
- Convert an audio file specifying which language the audio file is in to reduce conversion time:
```

- Specify where to save the converted files:

> `whisper audio.mp3 --output_dir "./output"`

Reviewer: ```suggestion
`whisper {{path/to/audio.mp3}} --output_dir "{{path/to/output}}"`
```

> - Specify where to save the converted files:

Reviewer 2: ```suggestion
- Convert an audio file and save it to a specific location:
```

- Change the verbose settings of whisper:

> `whisper audio.mp3 --verbose False`

Reviewer: ```suggestion
`whisper {{path/to/audio.mp3}} --verbose {{False}}`
```

> - Change the verbose settings of whisper:

Reviewer 2: ```suggestion
- Convert an audio file in quiet mode:
```
